### ISSUE TITLE
✅ 🐞 Bug Report: Dropdown items are not visible in the new editor Facelift When in Dark Mode

### ISSUE THREAD

**Issue by DaNTaF** [elementor/elementor#22366]
### Prerequisites

- [X] I have searched for similar issues in both open and closed tickets and cannot find a duplicate.
- [X] The issue still exists against the latest stable version of Elementor.

### Description

Text is not visible after version 3.12
version 3.13 still has this problem
grey text on grey background

Windows 11
Firefox + chrome

https://i.imgur.com/E4bJrNV.jpg

### Steps to reproduce

any dropdown menu on Elementor's backend 

### Isolating the problem

- [X] This bug happens with only Elementor plugin active (and Elementor Pro).
- [X] This bug happens with a Blank WordPress theme active ([Hello theme](https://wordpress.org/themes/hello-elementor/)).
- [ ] I can reproduce this bug consistently following the steps above.

### System Info

<details>
<summary>System Info</summary>

````txt


== Server Environment ==
	Operating System: Linux
	Software: Apache/2.4.56 (Debian)
	MySQL version: mariadb.org binary distribution v10.6.3
	PHP Version: 8.2.3
	PHP Memory Limit: 510M
	PHP Max Input Vars: 10000
	PHP Max Post Size: 100M
	GD Installed: Yes
	ZIP Installed: Yes
	Write Permissions: All right
	Elementor Library: Connected

== WordPress Environment ==
	Version: 6.2
	Site URL: 
	Home URL: 
	WP Multisite: No
	Max Upload Size: 100 MB
	Memory limit: 40M
	Max Memory limit: 510M
	Permalink Structure: /%postname%/
	Language: en-US
	Timezone: 0
	Admin Email: 
	Debug Mode: Inactive

== Theme ==
	Name: Hello Elementor
	Version: 2.7.1
	Author: Elementor Team
	Child Theme: No
== Features ==
	Custom Fonts: 0
	Custom Icons: 0

== Integrations ==
	


== Elementor Experiments ==
	Optimized DOM Output: Active by default
	Improved Asset Loading: Active by default
	Improved CSS Loading: Active by default
	Inline Font Icons: Inactive by default
	Accessibility Improvements: Active by default
	Additional Custom Breakpoints: Active by default
	admin_menu_rearrangement: Inactive by default
	Flexbox Container: Inactive by default
	Upgrade Swiper Library: Active by default
	Grid Container: Inactive by default
	Default to New Theme Builder: Active by default
	Hello Theme Header &amp; Footer: Active by default
	Editor Top Bar: Inactive by default
	Landing Pages: Active by default
	Nested Elements: Inactive by default
	Lazy Load Background Images: Inactive by default
	Page Transitions: Active by default
	Notes: Active by default
	Loop: Active by default
	Form Submissions: Active by default
	Scroll Snap: Active by default
	Menu: Inactive by default
````

</details>

**Comment by nicholaszein** [comment]
Hello @DaNTaF! 

Thank you for submitting your issue! 🙏 

We are aware of this issue and it will be addressed in a future version. The new Editor UI is still a work in progress, and we are gathering user feedback to better adapt it to our user's needs.

Best regards

**Comment by nickdgreen** [comment]
The Dark Interface is still very poor. The text editor options like the bulleted list icon become a white square when hovered over. and when active. The dropdown selections on, for example, the HTML tag for a title are a very pale grey on a white background (almost invisible). The options on, for example, a size setting (px, em, rem etc.) is a very pale pink again almost invisible. How is this better for the user?

**Comment by nicholaszein** [comment]
Hello there @DaNTaF!

We have great news! 🙌 

### 📢 We're happy to announce that the issue you raised was resolved in Elementor v3.14! 🥳 

✅ Feel free to check it out and update your plugin to the new version!

> _Check the changelog for more information:_
> [Changelog of Elementor and Elementor Pro](https://elementor.com/help/changelog-of-elementor/)

Cheers 🥂
